src/unix.rs

```diff
@@ -255,7 +257,10 @@ impl DrmWindowHandle {
     /// let handle = DrmWindowHandle::new(plane);
```

Comment:
Please update this example to also set the `connector_id`

Reply:
What I mean here is something like:

```suggestion
    /// # connector_id = unsafe { NonZeroU32::new_unchecked(1) };
    /// let mut handle = DrmWindowHandle::new(plane);
    /// handle.connector_id = Some(connector_id);
```